content/en/synthetics/dashboards/testing_coverage.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Application Testing Coverage Dashboard
+kind: documentation
+description: Learn about the out-of-the-box Synthetics application testing coverage dashboard.
+further_reading:
+- link: '/synthetics/browser_tests'
+  tag: 'Documentation'
+  text: 'Learn about Synthetic browser tests'
+- link: '/real_user_monitoring/'
+  tag: 'Documentation'
+  text: 'Learn about RUM & Session Replay'
+---
+
+## Overview
+
+The [application testing coverage dashboard][1] uses data collected from [RUM & Session Replay][2] and results from Synthetic [browser tests][3] to provide insights about the overall testing coverage for your application. 
+
+You can use this dashboard to build a more complete, accurate testing suite with information about the following topics:
+
+- A list of top web pages linked to RUM views
+- The percentage of tested RUM actions
+- The number of tested and total actions
+- The number of browser tests covering actions
+- The number of real user interactions 
+
+It shows:
+
+- **Untested actions**: Explore the most popular untested user actions with the count of real user interactions and the number of actions covered in browser tests.
+- **Top Synthetic browser tests**: Analyze the the list of browser tests with the count of tested RUM actions.
```

Comment:
here the top 2 things of the dashboard are:
- % of tested actions
- Untested actions.

I think we can remove the **Top Syn BT** mention as it brings less value.

Example of a sentence for % of tested actions
- **Coverage of tested actions**: Get at a glance the overall coverage of your application.

(we can do this change for both guides)